loops_labs.js

```diff
@@ -0,0 +1,223 @@
+// JOSEPH P. PASAOA
```

Comment:
Other than some minor feedback with if conditions (some conditions were unnecessary like in FizzBuzz question), good demonstration and implementation of loops. Keep up the good work!